cli/cmd/sync.go

```diff
@@ -163,10 +160,6 @@ func syncConnection(ctx context.Context, pm *plugins.PluginManager, sourceSpec s
 	}
 	_ = bar.Finish()
 	fmt.Println("Fetch completed successfully.")
-	fmt.Printf("Summary: Resources: %d, Failed Writes: %d, Fetch Errors: %d, Fetch Warnings: %d\n",
-		totalResources, failedWrites, sourcePlugin.Errors(), sourcePlugin.Warnings())
-	if sourcePlugin.Errors() > 0 || sourcePlugin.Warnings() > 0 || failedWrites > 0 {
-		fmt.Println("Please check the logs for more details on errors/warnings.")
-	}
```

Comment:
Why are these messages being removed, were they moved elsewhere?

Reply:
I need to open a follow-up PR but I want us to stop parsing logs and the plugins return number of error and or warnings as they have this information anyway, this will make also consistent experience between debug and release (which I know was quite weird before)

Reply:
Maybe keep these lines in there for now though, we can remove them in the follow-up?